server/models/User.js

```diff
@@ -5,21 +5,25 @@ const userSchema = new mongoose.Schema({
   username: {
     type: String,
     required: true,
-    unique: true
+    unique: true,
   },
   email: {
     type: String,
     required: true,
-    unique: true
+    unique: true,
   },
   password: {
     type: String,
-    required: true
+    required: true,
   },
   register_date: {
     type: Date,
-    default: Date.now
-  }
+    default: Date.now,
+  },
+  profile: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "profile",
```

Comment:
```suggestion
    ref: "profile",
    required: true
```

Nice work adding a ref here 👍🏽
Let's also make this a required attribute!